src/screens/AccountPin.js

```diff
@@ -96,6 +96,12 @@ class AccountPinView extends React.PureComponent {
     return (<Text style={styles.hintText}>Choose a PIN code with 6 or more digits</Text>)
   }
 
+  onPinInputChange = stateName => pinInput => {
```

Comment:
This is a weird notation that I'm not used to.
Any way to make it a little easier to read?